src/main/java/omoh/customexceptions/EmptyTaskNumberException.java

```diff
@@ -0,0 +1,4 @@
+package omoh.customexceptions;
+
+public class EmptyTaskNumberException extends Exception {
```

Comment:
I am not sure why the exception class here is completely empty? Shouldn't there be some kind of functionality?